form.py

```diff
@@ -0,0 +1,59 @@
+from flask_wtf import Form
+from wtforms import StringField, BooleanField, SelectField, TextAreaField, RadioField
+from flask_wtf.file import FileField
+from wtforms.validators import DataRequired, Email, URL
+from werkzeug import secure_filename
+
+class ApplicationForm(Form):
+    name = StringField('Name', validators=[DataRequired()])
```

Comment:
Yep, that's my next task. :)

On Thu, Sep 24, 2015 at 5:00 PM, Fureigh <notifications@github.com> wrote:

> In form.py
> <https://github.com/18F/joining-18f-app/pull/1#discussion_r40372547>:
>
> > @@ -0,0 +1,59 @@
> > +from flask_wtf import Form
> > +from wtforms import StringField, BooleanField, SelectField, TextAreaField, RadioField
> > +from flask_wtf.file import FileField
> > +from wtforms.validators import DataRequired, Email, URL
> > +from werkzeug import secure_filename
> > +
> > +class ApplicationForm(Form):
> > +    name = StringField('Name', validators=[DataRequired()])
>
> Errors aren't yet being communicated to the user. I'm cool with that being
> outside the scope of this PR, but let's get 'em in before launch.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/joining-18f-app/pull/1/files#r40372547>.
>



-- 
Alison Rowland
Technical Lead, https://fbopen.gsa.gov | 18F
202-317-0124